diff --git a/lib/commonjs/index.js b/lib/commonjs/index.js
index 783de6dd6cb375a480689bf69b8652f790f23e1a..a53a04e573e12f8c68bdfe3e367fe5a5ab7d36d2 100644
--- a/lib/commonjs/index.js
+++ b/lib/commonjs/index.js
@@ -1,51 +1,89 @@
 "use strict";
 
 Object.defineProperty(exports, "__esModule", {
-  value: true
+  value: true,
 });
+
+Object.defineProperty(exports, "createIconSourceCache", {
+  enumerable: true,
+  get: function () {
+    return _createIconSourceCache.default;
+  },
+});
+
 Object.defineProperty(exports, "DEFAULT_ICON_COLOR", {
   enumerable: true,
   get: function () {
     return _defaults.DEFAULT_ICON_COLOR;
-  }
+  },
 });
 Object.defineProperty(exports, "DEFAULT_ICON_SIZE", {
   enumerable: true,
   get: function () {
     return _defaults.DEFAULT_ICON_SIZE;
-  }
+  },
 });
 Object.defineProperty(exports, "createIconSet", {
   enumerable: true,
   get: function () {
     return _createIconSet.createIconSet;
-  }
+  },
+});
+
+Object.defineProperty(exports, "dynamicLoader", {
+  enumerable: true,
+  get: function () {
+    return _dynamicFontLoading.dynamicLoader;
+  },
 });
+Object.defineProperty(exports, "getImageSource", {
+  enumerable: true,
+  get: function () {
+    return _getImageSource.getImageSource;
+  },
+});
+Object.defineProperty(exports, "getImageSourceSync", {
+  enumerable: true,
+  get: function () {
+    return _getImageSource.getImageSourceSync;
+  },
+});
+
 Object.defineProperty(exports, "isDynamicLoadingEnabled", {
   enumerable: true,
   get: function () {
     return _dynamicLoadingSetting.isDynamicLoadingEnabled;
-  }
+  },
 });
 Object.defineProperty(exports, "isDynamicLoadingSupported", {
   enumerable: true,
   get: function () {
     return _dynamicLoadingSetting.isDynamicLoadingSupported;
-  }
+  },
 });
 Object.defineProperty(exports, "setDynamicLoadingEnabled", {
   enumerable: true,
   get: function () {
     return _dynamicLoadingSetting.setDynamicLoadingEnabled;
-  }
+  },
 });
 Object.defineProperty(exports, "setDynamicLoadingErrorCallback", {
   enumerable: true,
   get: function () {
     return _dynamicLoadingSetting.setDynamicLoadingErrorCallback;
-  }
+  },
 });
+
 var _createIconSet = require("./create-icon-set.js");
+var _createIconSourceCache = _interopRequireDefault(
+  require("./create-icon-source-cache.js"),
+);
 var _defaults = require("./defaults.js");
+var _dynamicFontLoading = require("./dynamicLoading/dynamic-font-loading.js");
 var _dynamicLoadingSetting = require("./dynamicLoading/dynamic-loading-setting.js");
+var _getImageSource = require("./get-image-source.js");
+
+function _interopRequireDefault(e) {
+  return e && e.__esModule ? e : { default: e };
+} 
 //# sourceMappingURL=index.js.map
diff --git a/lib/module/index.js b/lib/module/index.js
index 32dd3fea48c3180228bfda293b16bd0614359c6f..42825da309ac060a85a50b6eda9f2b0f66d9a9ab 100644
--- a/lib/module/index.js
+++ b/lib/module/index.js
@@ -2,5 +2,14 @@
 
 export { createIconSet } from "./create-icon-set.js";
 export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from "./defaults.js";
-export { isDynamicLoadingEnabled, isDynamicLoadingSupported, setDynamicLoadingEnabled, setDynamicLoadingErrorCallback } from "./dynamicLoading/dynamic-loading-setting.js";
+export {
+  isDynamicLoadingEnabled,
+  isDynamicLoadingSupported,
+  setDynamicLoadingEnabled,
+  setDynamicLoadingErrorCallback,
+} from "./dynamicLoading/dynamic-loading-setting.js";
+
+export { default as createIconSourceCache } from "./create-icon-source-cache.js";
+export { dynamicLoader } from "./dynamicLoading/dynamic-font-loading.js";
+export { getImageSource, getImageSourceSync } from "./get-image-source.js";
 //# sourceMappingURL=index.js.map
diff --git a/lib/typescript/commonjs/src/index.d.ts b/lib/typescript/commonjs/src/index.d.ts
index 1c9cff943c4501d286d888919868f6eb6b99b0cf..aaa8624793b043cd2d4e9819109edc66709a1957 100644
--- a/lib/typescript/commonjs/src/index.d.ts
+++ b/lib/typescript/commonjs/src/index.d.ts
@@ -1,5 +1,20 @@
-export type { CreateIconSetOptions, IconComponent, IconProps } from './create-icon-set';
-export { createIconSet } from './create-icon-set';
-export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from './defaults';
-export { isDynamicLoadingEnabled, isDynamicLoadingSupported, setDynamicLoadingEnabled, setDynamicLoadingErrorCallback, } from './dynamicLoading/dynamic-loading-setting';
+export type {
+  CreateIconSetOptions,
+  IconComponent,
+  IconProps,
+} from "./create-icon-set";
+export { createIconSet } from "./create-icon-set";
+export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from "./defaults";
+export {
+  isDynamicLoadingEnabled,
+  isDynamicLoadingSupported,
+  setDynamicLoadingEnabled,
+  setDynamicLoadingErrorCallback,
+} from "./dynamicLoading/dynamic-loading-setting";
+
+export type { FontSource } from "./dynamicLoading/types";
+export { default as createIconSourceCache } from "./create-icon-source-cache";
+export { dynamicLoader } from "./dynamicLoading/dynamic-font-loading";
+export { getImageSource, getImageSourceSync } from "./get-image-source";
+
 //# sourceMappingURL=index.d.ts.map
diff --git a/lib/typescript/module/src/index.d.ts b/lib/typescript/module/src/index.d.ts
index 1c9cff943c4501d286d888919868f6eb6b99b0cf..aaa8624793b043cd2d4e9819109edc66709a1957 100644
--- a/lib/typescript/module/src/index.d.ts
+++ b/lib/typescript/module/src/index.d.ts
@@ -1,5 +1,20 @@
-export type { CreateIconSetOptions, IconComponent, IconProps } from './create-icon-set';
-export { createIconSet } from './create-icon-set';
-export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from './defaults';
-export { isDynamicLoadingEnabled, isDynamicLoadingSupported, setDynamicLoadingEnabled, setDynamicLoadingErrorCallback, } from './dynamicLoading/dynamic-loading-setting';
+export type {
+  CreateIconSetOptions,
+  IconComponent,
+  IconProps,
+} from "./create-icon-set";
+export { createIconSet } from "./create-icon-set";
+export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from "./defaults";
+export {
+  isDynamicLoadingEnabled,
+  isDynamicLoadingSupported,
+  setDynamicLoadingEnabled,
+  setDynamicLoadingErrorCallback,
+} from "./dynamicLoading/dynamic-loading-setting";
+
+export type { FontSource } from "./dynamicLoading/types";
+export { default as createIconSourceCache } from "./create-icon-source-cache";
+export { dynamicLoader } from "./dynamicLoading/dynamic-font-loading";
+export { getImageSource, getImageSourceSync } from "./get-image-source";
+
 //# sourceMappingURL=index.d.ts.map
