diff --git a/lib/module/views/useHeaderConfigProps.js b/lib/module/views/useHeaderConfigProps.js
index efc24fc..313556c 100644
--- a/lib/module/views/useHeaderConfigProps.js
+++ b/lib/module/views/useHeaderConfigProps.js
@@ -85,12 +85,10 @@ const processBarButtonItems = (items, colors, fonts) => {
 };
 const transformIcon = icon => {
   if (icon?.type === 'image') {
-    return icon.tinted === false ? {
+    return {
       type: 'imageSource',
-      imageSource: icon.source
-    } : {
-      type: 'templateSource',
-      templateSource: icon.source
+      imageSource: icon.source,
+      renderingMode: icon.tinted === false ? 'original' : 'template'
     };
   }
   return icon;
