diff --git a/node_modules/@react-native-vector-icons/common/lib/module/index.js b/node_modules/@react-native-vector-icons/common/lib/module/index.js
index 32dd3fe..26676c0 100644
--- a/node_modules/@react-native-vector-icons/common/lib/module/index.js
+++ b/node_modules/@react-native-vector-icons/common/lib/module/index.js
@@ -3,4 +3,7 @@
 export { createIconSet } from "./create-icon-set.js";
 export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from "./defaults.js";
 export { isDynamicLoadingEnabled, isDynamicLoadingSupported, setDynamicLoadingEnabled, setDynamicLoadingErrorCallback } from "./dynamicLoading/dynamic-loading-setting.js";
+export { default as createIconSourceCache } from "./create-icon-source-cache.js";
+export { dynamicLoader } from "./dynamicLoading/dynamic-font-loading.js";
+export { getImageSource, getImageSourceSync } from "./get-image-source.js";
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@react-native-vector-icons/common/lib/typescript/module/src/index.d.ts b/node_modules/@react-native-vector-icons/common/lib/typescript/module/src/index.d.ts
index 1c9cff9..054ece0 100644
--- a/node_modules/@react-native-vector-icons/common/lib/typescript/module/src/index.d.ts
+++ b/node_modules/@react-native-vector-icons/common/lib/typescript/module/src/index.d.ts
@@ -2,4 +2,8 @@ export type { CreateIconSetOptions, IconComponent, IconProps } from './create-ic
 export { createIconSet } from './create-icon-set';
 export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from './defaults';
 export { isDynamicLoadingEnabled, isDynamicLoadingSupported, setDynamicLoadingEnabled, setDynamicLoadingErrorCallback, } from './dynamicLoading/dynamic-loading-setting';
+export type { FontSource } from './dynamicLoading/types';
+export { default as createIconSourceCache } from './create-icon-source-cache';
+export { dynamicLoader } from './dynamicLoading/dynamic-font-loading';
+export { getImageSource, getImageSourceSync } from './get-image-source';
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
